test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQServiceTests.cs

```diff
@@ -13,7 +13,7 @@ public class RabbitMQServiceTests
     {
         [Theory]
         [InlineData("", "localhost", "guest", "guest", 5672, "", "localhost", "guest", "guest", 5672)]
-        [InlineData("amqp://testUserName:testPassword@11.111.111.11:5672", null, null, null, null, "amqp://testUserName:testPassword@11.111.111.11:5672", "11.111.111.11", "testUserName", "testPassword", 5672)]
+        [InlineData("amqp://testUserName:testPassword@11.111.111.11:5672", null, null, null, 8080, "amqp://testUserName:testPassword@11.111.111.11:5672", "11.111.111.11", "testUserName", "testPassword", 5672)]

```

Comment:
was this meant to test null port?

Reply:
This stood out to me too. If this was indeed used to test null port, shouldn't GetConnectionFactory take a nullable int ?

Reply:
I think we default to a known port it not supplied. Can you check?

Reply:
Yeah, it comes from https://github.com/Azure/azure-functions-rabbitmq-extension/blob/dev/src/Bindings/RabbitMQClientBuilder.cs#L32. Looks like it defaults to 0, so I updated test for the default value.